### ISSUE TITLE
Unwanted capitalisation

### ISSUE THREAD

**Issue by Dan503** [hosseinmd/prettier-plugin-jsdoc#143]
When I write this:

```ts
export interface ConfirmationParams {
    /**
     * A title for the dialog
     *
     * @default 'Are you sure?'
     */
    title?: string
}
```

I get unwanted capitalisation in the `@default` tag. It says "Are **You** sure?" instead of "Are **you** sure?"

```ts
export interface ConfirmationParams {
    /**
     * A title for the dialog
     *
     * @default 'Are You sure?'
     */
    title?: string
}
```

**Comment by RedGuy12** [comment]
Yes, I also find this plugin to format `/** @file A file description */` as `/** @file A File description */`

**Comment by RishiKumarRay** [comment]
So we have to just capitalise the y? to solve the bug

**Comment by Dan503** [comment]
No the issue is that it is forcing a capital letter where there shouldn't be one. 

Correct English is:
"Are you sure?"

The bug is that it is forcing incorrect capitalisation on the 2nd word
"Are You sure?"

**Comment by Dan503** [comment]
I want it to just let me handle my own capitalisation of words in this situation.

**Comment by hosseinmd** [comment]
This module recognized `are` is default value and others are description, this is reason of capitalizing 2nd word.

**Comment by Dan503** [comment]
Can the capitalising of the first letter of the description feature just be removed from this plugin then?

I don't see it as an essential feature and I get the feeling that the code needed to know if it should or should not apply capitalisation would be super complicated and bug prone.

**Comment by hosseinmd** [comment]
Maybe we should do this with a configuration and set the default to false.

**Comment by hosseinmd** [comment]
false mean: do not capitalize

**Comment by Dan503** [comment]
It could be called `descriptionCapitalisation`

... might be a bit wordy though at least it's clear.

`desCaps` is shorter but less clear

**Comment by Dan503** [comment]
There is one thing that the plugin can do here to prevent some unwanted capitalisation.

```
@default 'Are you sure?'
```

I have quote marks around the text here. The plugin should be able to recognize that the full text inside the quote marks is the value and the text after the quote marks is the description.

```js
@default 'I am a value' I am the description
```

So I think the bug here is that it shouldn't capitalise text inside `'single quote'`, `"double quote"`, or `` `backtick quote` ``.

**Comment by Dan503** [comment]
Can you reopen the issue?

The quote marks issue stated in my previous comment is a valid bug.

**Comment by Dan503** [comment]
I tried setting this in my prettierrc file and it didn't work :(

```
"jsdocCapitalizeDescription": false
```

```json
{
    "printWidth": 120,
    "tabWidth": 4,
    "singleQuote": true,
    "semi": false,
    "jsxBracketSameLine": true,
    "arrowParens": "avoid",
    "jsdocParser": true,
    "jsdocCapitalizeDescription": false
}
```

**Comment by hosseinmd** [comment]
> There is one thing that the plugin can do here to prevent some unwanted capitalisation.
> 
> ```
> @default 'Are you sure?'
> ```
> 
> I have quote marks around the text here. The plugin should be able to recognize that the full text inside the quote marks is the value and the text after the quote marks is the description.
> 
> ```js
> @default 'I am a value' I am the description
> ```
> 
> So I think the bug here is that it shouldn't capitalise text inside `'single quote'`, `"double quote"`, or `` `backtick quote` ``.

Default tag has optional value if you want to declare value, you should keep it in `{here}` for example 
```
/**
 * @deffault {true} default value is true
 */

**Comment by hosseinmd** [comment]
This is not supported by vs code.

**Comment by hosseinmd** [comment]
If you have an object as default value, how you can add it to jsdoc?

**Comment by Dan503** [comment]
Objects are painful in jsDoc.

```js
/**
 * @param {Object} numberObject
 * @param {number} numberObject.number - the number to add 1 to
 */
const add1 = numberObject => numberObject.number + 1
```

**Comment by hosseinmd** [comment]
https://jsdoc.app/tags-default.html this is jsdoc default tag documention, there is no description

**Comment by hosseinmd** [comment]
In the other hand, VS Code rendering default value like example.

**Comment by hosseinmd** [comment]
> Yes, I also find this plugin to format `/** @file A file description */` as `/** @file A File description */`

I fixed this

**Comment by hosseinmd** [comment]
Fixed V0.3.33
